@here/harp-mapview/lib/Tile.ts

```diff
@@ -593,7 +593,7 @@ export class Tile implements CachedResource {
      * Called by [[VisibleTileSet]] to mark that [[Tile]] is visible and it should prepare geometry.
      */
     prepareForRender() {
-        if (!this.m_decodedTile || this.m_disposed) {
+        if (this.m_disposed) {
```

Comment:
Are you aware that `prepareForRendered` is called before EVERY frame for particular tile ?

And ...